src/app/events/event-components/event-participants/event-participants.component.ts

```diff
@@ -14,10 +15,14 @@ export class EventParticipantsComponent implements OnInit {
   defaultColumns: TableColumn[] = [];
   configKey = CONFIG_CASES;
   actionsBulkEdit: NavItem[] = actionsBulkEditDefs;
-
-  constructor(public eventParticipantService: EventParticipantService) {}
+  presetFilters: Filter[] = [];
+  constructor(public eventParticipantService: EventParticipantService, private route: Router) {}
 
   ngOnInit(): void {
+    const urlParts = this.route.url.split('/');
+    const eventId = urlParts[3];
+
+    this.presetFilters = [{ field: 'event', value: { uuid: eventId } }];
```

Comment:
this is not ok. It needs to have a subscribe for the url. And also, do not use url.split. Use the this.activatedRoute.params or something else

Reply:
Resolved.